### PR TITLE
Add a script to fetch apex automatically & build

### PR DESCRIPTION
**Pull request by floren**

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -36,4 +36,8 @@ or to run in terminal mode:
   ./util/GO9PTERM
 
 To netboot, follow the instructions here: https://github.com/Harvey-OS/harvey/wiki/Booting-Harvey-on-real-hardware-I-(TFTP)
+
+You may want to run the following to get APEX installed:
+
+	OS=linux ARCH=amd64 CC=gcc sh getapex.sh
 EOF
```
Comment (rminnich) [inline bootstrap.sh:43]: does this mean you are missing a newline at end of file?

File: usr/harvey/lib/profile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 bind -a $home/bin/rc /bin
 bind -a $home/bin/$cputype /bin
+bind -a /apex/amd64/bin /bin
```
Comment (rminnich) [inline usr/harvey/lib/profile:3]: if this bind fails will the script break and stop here?

**Review by gmacd — DISMISSED** [review]

File: usr/harvey/lib/profile (hunk 2 above)
Comment (floren) [reply usr/harvey/lib/profile:3]: No, I've tried it; an error is printed, but the script continues

File: bootstrap.sh (hunk 1 above)
Comment (floren) [reply bootstrap.sh:43]: Does what mean? That's a literal "EOF" string, it's a here document

**Comment by floren** [comment]
ptal

**Review by rminnich — APPROVED** [review]

**Pull request merged**